workflows/scripts/normalize.py

```diff
@@ -40,36 +40,27 @@ def normalization(
 
     # loading parameters
     logging.info(f"Loading Annotation configuration from: {config}")
+    normalize_config = config["params"]
 
-    normalize_obj = Path(config)
-    normalize_config_path = normalize_obj.absolute()
-    if not normalize_obj.is_file():
-        e_msg = "Unable to find Normalization configuration file"
-        logging.error(e_msg)
-        raise FileNotFoundError(e_msg)
-
-    with open(normalize_config_path, "r") as yaml_contents:
-        normalize_config = yaml.safe_load(yaml_contents)["normalize_configs"]["params"]
-        logging.info("Annotation configuration loaded")
-
-    meta_features = [
-        "Metadata_Plate",
-        "Metadata_Well",
-        "Metadata_WellRow",
-        "Metadata_WellCol",
-        "Metadata_gene_name",
-        "Metadata_pert_name",
-        "Metadata_broad_sample",
-        "Metadata_cell_line",
-    ]
+    # -- this is form old version of CellProfiler
+    # meta_features = [
+    #     "Metadata_Plate",
+    #     "Metadata_Well",
+    #     "Metadata_WellRow",
+    #     "Metadata_WellCol",
+    #     "Metadata_gene_name",
+    #     "Metadata_pert_name",
+    #     "Metadata_broad_sample",
+    #     "Metadata_cell_line",
+    # ]
```

Comment:
Consider removing these comments or adding it into more formal documentation somehow.

Reply:
Removed! I think this can just be added into the config file if needed